### PR TITLE
Update modDesc.xml

### PR DESCRIPTION
**Pull request by vitalii2011**

### CODE REVIEW

File: modDesc.xml [hunk 1]
```diff
@@ -5,7 +5,7 @@
     <title>
         <en>Target Fill Level</en>
         <de>Ziel-Füllstand</de>
-        <ru>Уровень Заполнения Транспорта</ru>
+        <ru>Уровень Заполнения ТС И Хранилищ</ru>
```
Comment (schliesser) [inline modDesc.xml:8]: As a non russian speaker: what does TC stand for and is this a well known abbreviation?

**Review by schliesser — COMMENTED** [review]
Heyho @vitalii2011 

thank you very much for your contribution!
Can you please clarify the TC question for me?
If you prefer you can also answer in german :-)

**Comment by vitalii2011** [comment]
ТС - Транспортное средство (de., Fahrzeug

https://www.drive2.ru/b/1678619/
"ТС — сокращение словосочетания «транспортное средство». 
Используется часто, в том числе и в нормативных документах.".
"TS ist eine Abkürzung für den Begriff „Fahrzeug“.
Es wird häufig verwendet, auch in Regulierungsdokumenten".

**Review by schliesser — APPROVED** [review]

**Comment by schliesser** [comment]
Thank you very much for the clarification! It's merged.
Don't know when I'm able to create a new release.

**Pull request merged**